groups/templates/groups/file_comment.html

```diff
@@ -1,5 +1,9 @@
 {% extends "groups/comment_base.html" %}
+{% load static %}
 
 {% block comment_visible %}
-    <p>{{ comment.file }}</p>
+    <p>
+        Uploaded a file:
+        <a href="{% get_media_prefix %}{{ comment.file }}">{{ comment.file.name|slice:"21:" }}</a>
```

Comment:
I don't know a better way to get rid of the `/groups/file_comments/` part of the path :P There don't appear to be any suitable filters.